build/webpack.dist.js

```diff
@@ -18,46 +17,15 @@ const config = merge(base, {
     },
 
     plugins: [
+        new MiniCssExtractPlugin({
+            filename: options.isProduction ? 'keen-ui.min.css' : 'keen-ui.css'
+        }),
         new webpack.BannerPlugin({
             banner: options.banner,
             raw: true,
             entryOnly: true
-        }),
-
-        new ExtractTextPlugin({
-            filename: options.isProduction ? 'keen-ui.min.css' : 'keen-ui.css'
         })
     ]
-}, { clone: false });
-
-// First item in module.rules array is Vue
-config.module.rules[0].options.loaders = {
-    scss: ExtractTextPlugin.extract({
-        loader: 'css-loader!sass-loader',
-        fallbackLoader: 'vue-style-loader'
-    })
-};
-
-if (options.isProduction) {
-    config.plugins = config.plugins.concat([
-        new webpack.LoaderOptionsPlugin({
-            minimize: true
-        }),
-
-        // Set the production environment
-        new webpack.DefinePlugin({
-            'process.env': {
-                NODE_ENV: '"production"'
-            }
-        }),
-
-        // Minify with dead-code elimination
-        new webpack.optimize.UglifyJsPlugin({
-            compress: {
-                warnings: false
-            }
-        })
-    ]);
-}
+}, true);
```

Comment:
We could make this take an options object so the option name provides context for what the boolean is without the need to check the function definition. Same applies for the other webpack config files.

```suggestion
}, { extractCss: true });
```

Reply:
Sounds good.